src/popup/TolgeeDetector.tsx

```diff
@@ -1,41 +1,167 @@
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 15,
+          width: 400,
+        }}
+      >
+        <Typography variant="subtitle1" style={{ paddingBottom: 8 }}>
+          Tolgee settings
+        </Typography>
+        <TextField
+          label="API url"
+          variant="outlined"
+          value={values?.apiUrl || ''}
+          onChange={(e) =>
+            dispatch({
+              type: 'CHANGE_VALUES',
+              payload: { apiUrl: e.target.value },
+            })
+          }
+          onKeyDown={handleKeyDown}
+          size="small"
+          disabled={disabled}
+        />
+        <FormControl>
+          <TextField
+            label="API key"
+            variant="outlined"
+            value={values?.apiKey || ''}
+            onChange={(e) =>
+              dispatch({
+                type: 'CHANGE_VALUES',
+                payload: { apiKey: e.target.value },
+              })
+            }
+            onKeyDown={handleKeyDown}
+            size="small"
+            disabled={disabled}
+          />
+          <FormHelperText
+            error={credentialsCheck === 'invalid'}
+            style={{ height: 15 }}
+          >
+            {credentialsCheck === null ? (
+              ''
+            ) : credentialsCheck === 'loading' ? (
+              '...'
+            ) : credentialsCheck === 'invalid' ? (
+              'Invalid'
+            ) : (
+              <span style={{ color: 'green' }}>Valid</span>
+            )}
+          </FormHelperText>
+        </FormControl>
+        {!disabled && (
+          <Box display="flex" justifyContent="space-between">
+            <Box display="flex" style={{ gap: 5 }}>
+              {dataPresent && (
+                <>
+                  <Switch
+                    size="small"
+                    checked={Boolean(appliedValues)}
+                    onChange={handleApplyChange}
+                    color="primary"
+                  />
+                  <Typography>Applied</Typography>
+                </>
+              )}
+            </Box>
+            <Box display="flex" style={{ gap: 10 }}>
+              {dataPresent && (
+                <Button
+                  size="small"
+                  onClick={() => dispatch({ type: 'CLEAR_ALL' })}
+                  variant="contained"
+                >
+                  Clear
+                </Button>
+              )}
+              <Button
+                size="small"
+                onClick={() => dispatch({ type: 'APPLY_VALUES' })}
+                variant="contained"
+                color="primary"
+                disabled={!validateValues(values)}
+              >
+                Apply
+              </Button>
+            </Box>
+          </Box>
+        )}
+      </Box>
+    );
+  } else if (tolgeePresent === 'legacy') {
     return (
-      <Box width="300px" p={2}>
-        This website is using Tolgee!
+      <Box width="400px" p={1}>
```

Comment:
400px can be const, it's repeated there 3 times

Reply:
Ok